pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -3875,13 +3875,23 @@ func (ms *MutexStatus) LockWaiting(holderKey, lockKey string, currentHolders []s
 	return false
 }
 
-func (ms *MutexStatus) LockAcquired(holderKey, lockKey string, currentHolders []string) bool {
-	i, mutexHolding := ms.GetHolding(lockKey)
+// GetHoldingName returns holding names without
+// any splitting of strings.
+func GetHoldingName(holderKey string) string {
```

Comment:
This function doesn't really do anything, shouldn't we drop it to aid readability?